prime-router/src/main/resources/metadata/hl7_mapping/resources/ServiceRequest/OBRExtension.yml

```diff
@@ -16,6 +16,28 @@ elements:
     constants:
       eiFieldPath: '%{obrFieldPath}-3'
 
+  - name: observation-collected-datetime
```

Comment:
Just a note here to take care that any OBR mappings that populate these fields don't get overwritten by the extension value here.